projects/igniteui-angular/src/lib/tabs/tabs-group.component.ts

```diff
@@ -23,46 +23,125 @@ import { IgxTabItemTemplateDirective } from './tabs.directives';
 })
 
 export class IgxTabsGroupComponent implements AfterContentInit, AfterViewChecked {
+    private _itemStyle = 'igx-tabs-group';
+    /**
+     * @hidden
+     */
     public isSelected = false;
 
+    /**
+    * An @Input property that sets the value of the `label`.
+    *```typescript
+    *<igx-tabs-group label="Tab 1" icon="folder">
+    *```
+    */
     @Input()
     public label: string;
 
+    /**
+    * An @Input property that sets the value of the `icon`.
```

Comment:
@alexalex972 Should this be a valid icon name from the icon set? Or is this a title of the icon? We should clarify.